src/Components/Chakra__Select.res

```diff
@@ -0,0 +1,803 @@
+open Chakra__MakeProps
+
+@react.component @module("@chakra-ui/react")
+external make: (
+  ~children: React.element=?,
+  /**
+    Component Props {Select}
+   **/
```

Comment:
9 to 23 are the only ones I made, the rest were generated from the Box template as per instructions.

Reply:
Nice, thanks for your contribution.

Can you change type of props size and variant, use polyvariant instead of "string". You can see in `interfaces/Chakra__Theme.res` for make a type and  be use in this file.

Reply:
Fixed.

I wasn't exactly sure about makeProps, but it seems like it needs a def in `Chakra__MakeProps.res`, at least when I looked at the other examples of components using their own polyvariants.

Reply:
Yes, will be refactor and make it single source of definition, soo! Maybe I will make a roadmap for make it clear for all contributor. 

Thanks for your advice @tjbo